sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -769,6 +769,44 @@ export interface BlobProperties {
     tagCount?: number;
 }
 
+// @public
+export interface BlobQueryCsvTextConfiguration extends BlobQueryTextConfiguration {
+    columnSeparator: string;
+    escapeCharacter?: string;
+    fieldQuote?: string;
+    hasHeaders: boolean;
+    kind: "csv";
+}
+
+// @public
+export interface BlobQueryError {
```

Comment:
Can you give an example of errors that occur here, both fatal and non-fatal?

Reply:
@bterlson Pinged you examples in another 2 review comments

Reply:
I see, thanks.